app/models/episode.rb

```diff
@@ -336,4 +342,11 @@ def sanitize_text
   def feeder_cdn_host
     ENV['FEEDER_CDN_HOST']
   end
+
+  def full_title
```

Comment:
I think folks may have different ideas about how to include these values for their show.

so one way to go is add a truly new field, just for an itunes title (which feels duplicative, so I was originally against it)

or we may need to offer some kind of template so shows can have their own format for how to add the season and episode numbers to their titles.

e.g.
99% invisible: "247- Usonia the Beautiful"
template: "{episode_number}- {title}"

twww: "0.04: Big Block of Cheese III (with 99% Invisible)"
template: "{episode_number}.{season}: {title}"
(that is season 4, episode 0 by the way)